components/ai_chat/core/common/mojom/page_content_extractor.mojom

```diff
@@ -13,10 +13,17 @@ enum PageContentType {
   VideoTranscriptVTT
 };
 
+// Configuration for YouTube InnerTube API
+struct InnerTubeConfig {
```

Comment:
I don't care if we keep `InnerTube` or not because I don't know if it's really important what YT calls it, but I think we should include `YouTube` in the name

Reply:
renamed in f8c513490d6604ad75f5f40b7d93146029369b53 and 81a9193d389aefb31e9980632d27a694c458a262